tests/handlers/labs_test.py

```diff
@@ -560,7 +586,7 @@ async def test_spawn_errors(
     mock_slack: MockSlackWebhook,
 ) -> None:
     lab = read_input_lab_specification_json("base", "lab-specification.json")
-    apis_to_fail = {"read_namespaced_secret"}
+    apis_to_fail = set()
```

Comment:
Is this change unrelated?  If not, I don't understand what reading a namespaced secret has to do with it.

Reply:
Sorry, that was me being lazy. Unrelated (but correct) change left over from testing something else.